### DISCUSSION TITLE
Threshold Probability vs Model Cutoff vs Harm: What is the difference?

### DISCUSSION THREAD

**Discussion by tuckerparon** [MSKCC-Epi-Bio/decisioncurveanalysis#13] [Q&A]
Hello everybody,

I'm having some trouble wrapping my head around the difference between a couple key concepts in DCA. Below are my main questions with some context.

**1. What is the difference, if any, between the threshold values and the probability cutoff employed onto the model?**

If a model utilizes a cutoff of .1 to determine the binary outcome, should the model designers focus on the net benefit where the threshold value is .1? And should the modelers pass in their outcome  variable as a binary created by the .1 cutoff, or pass it in as a continuous and focus on the threshold of .1, or is that the wrong way of thinking of it alltogether. It's my understanding that using the DCA plot to select a threshold value for the model is WRONG ([7 Common Errors in DCA](https://www.fharrell.com/post/edca/))), but I'm just confused on the difference between threshold and cutoff. I've read the original paper, the documentation, and several studies utilizing DCA, but am still quite stuck. 

**2. Similarly, how is the harm variable different than threshold probability?**

From [the 2006 paper](https://pubmed.ncbi.nlm.nih.gov/17099194/), "If a clinician or a patient thought that missing a case of disease was 50 times worse than having to undergo testing, the test harm would be rated as 0.02." How is this different than the concept of threshold probability (comparing harm of treatment against harm of disease)?

**Comment by VickersA** [comment]
1. The cut-off is the same as the threshold probability. The key idea of decision curve analysis is that we use the same number (e.g. 10%) to determine who is positive vs. negative and to weight the consequences of true and false positives. With respect to "If a model utilizes a cutoff of .1 to determine the binary outcome, should the model designers focus on the net benefit where the threshold value is .1?", there are two answer: a) yes; b) often irrelevant, because there models don't generally need to be converted to a binary outcome by the model designers, the model should report the probability and let the user decided. 
2. The harm variable is actually "test harm" and is related to the harm required to get the information to run the model. Threshold probability is related to the harm of a false positive. Let's say, for instance, I have a model to predict whether someone has appendicitis and needs their appendix removed. I choose the threshold based on the harms of removing someone's appendix unnecessarily versus the harms of not resecting and the patients appendix bursting. Now let's say that my model required a painful or invasive test (e.g. a biopsy) in order to get a variable that was in the model. That would be test harm.

**Reply by tuckerparon** [reply]
Hi @VickersA - thanks so much for this response and apologies for not replying sooner. I am set on Question 2, but the threshold probability and cutoff relationship is still confusing me. If they are the same, how can threshold probability incorporate harm of treatment? I would define cutoff as the % risk of disease at which it would be unwise not to treat, but it seems like probability threshold is more nuanced. 

* The disconnect between threshold and cutoff is most evident for me when reading the threshold interpretations:
  * "This threshold probability, pt, is where the expected benefit of treatment is equal to the expected benefit of avoiding treatment" ([Original 2006 Paper](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2577036/pdf/nihms74373.pdf))
  * ie. (using the prostate cancer example) A 10% threshold would mean we are willing to biopsy no more than 10 men to find one case of prostate cancer. So the benefit of detecting one prostate cancer is 9 times the harm of an unnecessary biopsy (1:9 harm-to-benefit ratio). 
    * This seems marginally different than a model cutoff which seems just to consider risk of disease (and not risk of treatment).

* Additionally, the threshold seems more to do with the patient's opinion (whereas cutoff would be more clinician based?):
   * "the threshold probability of a disease or event at which a patient would opt for treatment is informative of how the patient weighs the relative harms of a false-positive and a false-negative prediction" ([Original 2006 Paper](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2577036/pdf/nihms74373.pdf))

Lastly, I was curious about the case-control implementation.
* Would it be appropriate to use case-control if the two population groups were high and low risk of that condition? I assume this would be appropriate, as this would correct for the population prevalence as is outlined in the DCA tutorial?

**Comment by VickersA** [comment]
I do suggest you do more reading, perhaps looking at some of the introductory papers on the website (e.g. BMJ paper, "step by step" guide etc). Cut off and threshold are the same idea and consider both harm of an intervention and harm of the disease if no treatment. Threshold vs. cutoff are not to do with doctor vs. patient; we talk generally off a "decision maker". With respect to case control, this depends on a definition of "case-ness" and I have not seen a definition of case in terms of "high risk"
